content/docs/querying/functions.md

```diff
@@ -336,3 +325,4 @@ over time and return an instant vector with per-series aggregation results:
 * `max_over_time(range-vector)`: the maximum value of all points under the specified interval.
 * `sum_over_time(range-vector)`: the sum of all values under the specified interval.
 * `count_over_time(range-vector)`: the count of all values under the specified interval.
+* `quantile_over_time(scalar, range-vector)`: the given quantile of the values under the specified interval.
```

Comment:
As we did in the code itself, should we change "under" to a less weird preposition like "in" or "within" or "contained in"?